coordinator/src/utils/status-code.ts

```diff
@@ -0,0 +1,11 @@
+import {fhir} from "@models"
+
+export function getStatusCode(issues: Array<fhir.OperationOutcomeIssue>): number {
+  if (issues.every(i => i.code === fhir.IssueCodes.INFORMATIONAL)) {
+    return 200
+  }
+  if (issues.every(i => i.code === fhir.IssueCodes.FORBIDDEN)) {
+    return 403
+  }
+  return 400
```

Comment:
It would be useful to add a log message here so we can see which IssueCodes were not explicitly handling

Reply:
It will be nearly all of them, but intentionally so

Reply:
Will we do anything with this logging?

Reply:
not sure this would really do anything since we do handle all codes explicitly. this is just letting us determine which status code to use for a certain error type